### PR TITLE
Add initial Landing page

### PR DESCRIPTION
**Pull request by cpanato**
We can change later :)

 - fixes https://github.com/vmware-tanzu/helm-charts/issues/16

/cc @carlisia

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# VMware Tanzu Helm Repository
+
+![VMware Tanzu](assets/vmware-tanzu-logo.png)
+
+Add VMware Tanzu repository to Helm repos:
+
+```bash
+helm repo add vmware-tanzu https://vmware-tanzu.github.io/helm-charts
+```
+
```
Comment (carlisia) [inline index.md]: Let's add ## Charts

File: index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+## Install Velero
```
Comment (carlisia) [inline index.md]:
```suggestion
### Velero
```

File: index.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
```
Comment (carlisia) [inline index.md]:
```suggestion
To install Velero:
```

File: index.md (hunk 1 above)
Comment (cpanato) [reply index.md]: PTAL @carlisia

File: index.md (hunk 2 above)
Comment (cpanato) [reply index.md]: done

File: index.md (hunk 3 above)
Comment (cpanato) [reply index.md]: done

**Comment by carlisia** [comment]
@nbr PTAL.

**Comment by nrb** [comment]
@carlisia Wrong person fyi ;)

**Comment by carlisia** [comment]
I'm dyslexic apparently 😭

**Pull request merged**